operator/apis/loki/v1beta1/lokistack_types_test.go

```diff
@@ -0,0 +1,1061 @@
+package v1beta1_test
+
+import (
+	"testing"
+
+	v1 "github.com/grafana/loki/operator/apis/loki/v1"
+	"github.com/grafana/loki/operator/apis/loki/v1beta1"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestConvertToV1(t *testing.T) {
+	tt := []struct {
+		desc string
+		src  v1beta1.LokiStack
+		dst  v1.LokiStack
```

Comment:
I don't think `dst` will ever be filled with any "data", only GroupVersionKind information will be set, looking at the base [webhook code](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.11.0/pkg/webhook/conversion/conversion.go#L204). I think I would always initialize it empty and leave it out of the test table in that case.